io.openems.edge.goodwe.et/readme.md

```diff
@@ -0,0 +1,8 @@
+# io.openems.edge.goodwe.et Provider
```

Comment:
"readme.md" file is not used anymore. Please provide "readme.adoc" file; take other bundles as reference.